pkg/client/cli/util/connector.go

```diff
@@ -58,7 +56,7 @@ func UserDaemonDisconnect(ctx context.Context, quitDaemons bool) (err error) {
 		fmt.Fprint(stdout, "quitting...")
 	} else {
 		fmt.Fprint(stdout, "disconnecting...")
-		if _, err = ud.Disconnect(ctx, &empty.Empty{}); status.Code(err) != codes.Unimplemented {
+		if _, err = ud.Disconnect(ctx, &empty.Empty{}); grpcStatus.Code(err) != grpcCodes.Unimplemented {
```

Comment:
This is harmless, but what motivates this change?

Reply:
I did this one, neatness, a lib was imported twice